src/TritonCTS/src/DbWrapper.cpp

```diff
@@ -108,12 +111,12 @@ void DbWrapper::initAllClocks() {
         }
 
         if (getNumClocks() <= 0) {
-                std::cout << "\n";
-                std::cout << " [ERROR] No clock nets have been found.\n";
-                std::exit(1);
+                std::string errorMsg = "No clock nets have been found.\n";
+                error(errorMsg.c_str());
```

Comment:
what is the point of creating an std::string constant and then discarding it?
just pass the constant string in to the error function